src/plugins/jingleNodes/src/java/org/jinglenodes/JingleNodesPlugin.java

```diff
@@ -40,6 +40,9 @@
 import org.xmpp.jnodes.nio.PublicIPResolver;
 
 public class JingleNodesPlugin implements Plugin {
+	
+	private static final String[] 	stun_host = 	{"stun.l.google.com",	"stun.schlund.de",	"stun.iptel.org ",	"stun4.l.google.com"};
```

Comment:
I think there should be a method in the core to get the external IP address and information about the reachability. Plugins could then query this information. 
Sadly,  I am not familiar enough with Openfire to implement it easily. 
I just needed a fix for the Jingle plugin.